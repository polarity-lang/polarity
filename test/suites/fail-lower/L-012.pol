data Bool { True, False }

data Option(_: Type) {
    None(elem: Type): Option(elem),
    Some(elem: Type, x: elem): Option(elem),
}

let example2: Option(wrong := Bool) { None(Bool) }
