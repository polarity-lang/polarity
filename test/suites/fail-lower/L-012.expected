L-012

  × Used named argument wrong for wildcard parameter
   ╭─[L-012.pol:2:1]
 2 │ 
 3 │ data Option(_: Type) {
   ·             ─
 4 │     None(elem: Type): Option(elem),
   ╰────
